velox/docs/develop/task.rst

```diff
@@ -0,0 +1,256 @@
+===================
+What's in the Task?
+===================
+
+**TL;DR** Some operators need to coordinate their work with other operators. Table
+scan operators running in parallel need to pull splits from a shared queue. The
+HashProbe operator needs to wait for the HashBuild operator to create a hash
+table. Multiple HashBuild operators building different sections of the hash
+table in parallel need to coordinate to combine the pieces into one final hash
+table. Exchange operators running in parallel need to pull data from shared
+remote sources. Local exchange operators need to write and read data into
+shared data partitions. Task is maintaining shared state and enabling
```

Comment:
Adding the following makes it sounds better to me `Task coordinates these by maintaining a shared state .....`